tests/test_parser_delete.py

```diff
@@ -19,14 +19,10 @@ def setUpClass(cls):
         cls.parser_under_test, manager, mock_command = initialize_test_pieces(commandname)
         DeleteParser.delete_parser(manager, mock_command)
 
-    def test_delete_parser_no_object(self):
```

Comment:
this should still throw - it might need an exclusive argument group?

Reply:
Thanks, Thats true, I think I was missing the either/or case when using exclusive arg group.